pymc3/stats.py

```diff
@@ -532,7 +538,7 @@ def gradient(w):
                                round(weight, round_to),
                                round(se, round_to),
                                round(d_se, round_to),
-                               warns[idx])
```

Comment:
I think you need to revert this.

Reply:
Yes, there is some more bookkeeping necessary to get that part right.